test/rest.test.js

```diff
@@ -2253,6 +2276,8 @@ describe('strong-remoting-rest', function() {
   });
 
   it('rejects multi-item array passed to a number argument', function(done) {
+    /* it rejects multi-item array passed to integer argument
+     as it coerces to number type */
```

Comment:
yes, IIRC. As it will be an exact similar code block, will include the test case for `integer`